examples/EMTemplates/EM_GeneralPerfExperiments_Template.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "experiment_name": "quarkus-resteasy-autotune-min-http-response-time-db",
+    "resource": {
+      "namespace": "Default",
+      "deployment_name": "tfb-qrh-sample"
+    },
+    "datasource_info": {
+      "prometheus1": {
+        "url": "http://10.101.144.137:9090",
+        "provider": "prometheus"
+      }
+    },
+    "settings": {
+      "trial_settings": {
+        "measurement_cycles": "3",
+        "warmup_duration": "1min",
+        "warmup_cycles": "3",
+        "measurement_duration": "1min",
+        "iterations": "3"
+      },
+      "deployment_settings": {
+        "deployment_tracking": {
+          "trackers": [
+            "training"
+          ]
+        },
+        "deployment_policy": {
+          "type": "rollingUpdate"
+        }
+      }
+    },
+    "metrics": {
+      "pod_metrics": {
+        "request_sum": {
+          "datasource": "prometheus1",
+          "query": "rate(http_server_requests_seconds_sum{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
+          "name": "request_sum"
+        },
+        "request_count": {
+          "datasource": "prometheus1",
+          "query": "rate(http_server_requests_seconds_count{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
+          "name": "request_count"
+        }
+      },
+      "container_metrics": {
```

Comment:
Earlier version of EM input JSON has structured the container metrics in a array format as there might be multiple containers per pod. With the current approach that use case might be missed or we need to make EM understand the new input JSON way of detecting multiple containers. Something like passing container name as one of the attribute.

Reply:
done